### ISSUE TITLE
Events not bubbling

### ISSUE THREAD

**Issue by aboba** [w3c/webrtc-pc#1273]
From EKR: https://lists.w3.org/Archives/Public/public-webrtc/2017May/0166.html

A similar comment about pulling stuff out is with the boilerplate
about events not bubbling or being cancelable.

**Comment by aboba** [comment]
There are four instances in the specification with events "which does not bubble (except where otherwise stated) and is not cancelable (except where otherwise stated)".  Does it make sense to pull this text out, or just leave it as it is?

**Comment by stefhak** [comment]
Four cases is not that much, I think we could leave that as is.

**Comment by jan-ivar** [comment]
Yes we seem to be inconsistent here at least. Searching for "Fire", some places, like tonechange event, is a sub-algorithm that talks about bubbling, whereas many others like "RTCError event" does not.

**Comment by stefhak** [comment]
Sorry for my ignorance, but what reasons are we using to select whether an event should:
- fire as a simple event (no wording about bubbling or being cancable)
- fire as an event (no wording about bubbling or being cancable)
- fire as an event that does not bubble and is not cancable?

**Comment by alvestrand** [comment]
Plan: Remove all text about bubbling and cancelling, and refer to a DOM spec that defaults to "not bubble and not cancellable".

**Comment by stefhak** [comment]
@aboba I think this can be closed given the merge of #1966.
